proto/s2s.go

```diff
@@ -65,9 +72,10 @@ type RequestArg struct {
 // JobLog represents a log entry for a finished job.
 type JobLog struct {
 	// These three fields uniquely identify an entry in the job log.
-	RequestId string `json:"requestId"`
-	JobId     string `json:"jobId"`
-	Try       uint   `json:"try"` // try number N of 1 + Job.Retry
+	RequestId   string `json:"requestId"`
+	JobId       string `json:"jobId"`
+	Try         uint   `json:"try"`         // try number N of 1 + Job.Retry
```

Comment:
Now that we have multiple "try", let's disambiguate this one: `JobTry`.

Reply:
I think this rename is best to do in another PR. See my comment here: https://github.com/square/spincycle/pull/43#discussion_r172871963